mqr/views.py

```diff
@@ -3,11 +3,50 @@
 from rest_framework import generics, permissions
 from rest_framework.response import Response
 
+from .models import MqrStrata
+from .serializers import MqrStrataSerializer
+
+arms = ["ARM", "RCM", "BCM", "RCM_BCM", "RCM_SMS"]
```

Comment:
Lets change this variable name to uppercase and maybe a bit more descriptive like STUDY_ARMS